### PR TITLE
Module 8 example finished, Images to be finished

### PR DESCRIPTION
**Pull request by PINGXIANG**

### CODE REVIEW

File: problemsets-mat223/modules/module8.tex [hunk 1]
```diff
@@ -23,7 +23,10 @@
 	where $\vec b_1=\xhat+\yhat$, and $\vec b_2 =3\yhat$ be another basis for $\R^2$. Given that $\vec v=2\xhat-\yhat$, 
 	find $[\vec v]_{\mathcal E}$ and $[\vec v]_{\mathcal B}$.
 
-	XXX Finish
+	Given that $\vec v=2\xhat-\yhat=2(\xhat+\yhat)-3\yhat=2\vec b_1-3\vec b_2$, we could find
```
[the remark this answers is not in the document]
Comment (PINGXIANG) [reply problemsets-mat223/modules/module8.tex]: I see. But shall I do it by saying set v = xb_1 +yb_2? The definition of the linear combination is in module 9. Here I guess students, if they don't preview, would not know what is linear combination.

[the remark this answers is not in the document]
Comment (PINGXIANG) [reply problemsets-mat223/modules/module8.tex]: Ok never mind, my memory goes around modules 6 to 9

File: problemsets-mat223/modules/module8.tex [hunk 2]
```diff
@@ -23,7 +23,30 @@
 	where $\vec b_1=\xhat+\yhat$, and $\vec b_2 =3\yhat$ be another basis for $\R^2$. Given that $\vec v=2\xhat-\yhat$, 
 	find $[\vec v]_{\mathcal E}$ and $[\vec v]_{\mathcal B}$.
 
-	XXX Finish
+	Given that $\vec v=2\xhat-\yhat$, we could find
```
Comment (siefkenj) [inline problemsets-mat223/modules/module8.tex]: `could find` -> `see`.

File: problemsets-mat223/modules/module8.tex [hunk 3]
```diff
@@ -23,7 +23,30 @@
 	where $\vec b_1=\xhat+\yhat$, and $\vec b_2 =3\yhat$ be another basis for $\R^2$. Given that $\vec v=2\xhat-\yhat$, 
 	find $[\vec v]_{\mathcal E}$ and $[\vec v]_{\mathcal B}$.
 
-	XXX Finish
+	Given that $\vec v=2\xhat-\yhat$, we could find 
+	\[
+	    [\vec v]_{\mathcal E}=\mat{2\\-1}.
+	\]
+	To find $[\vec v]_{\mathcal B}$, we could use a system of equations to find $\vec v$ as a linear combination of $\vec b_1$ and $\vec b_2$. That is, set
```
Comment (siefkenj) [inline problemsets-mat223/modules/module8.tex]: `could` -> `can`.  "could" indicates a counterfactual. "We *could* do it this way, but we won't. We'll do it this other way." However, we are doing it this way, so you say "We *can* do it this way"

File: problemsets-mat223/modules/module8.tex [hunk 4]
```diff
@@ -23,7 +23,30 @@
...
-	XXX Finish
+	Given that $\vec v=2\xhat-\yhat$, we could find 
+	\[
+	    [\vec v]_{\mathcal E}=\mat{2\\-1}.
+	\]
+	To find $[\vec v]_{\mathcal B}$, we could use a system of equations to find $\vec v$ as a linear combination of $\vec b_1$ and $\vec b_2$. That is, set 
+	\[
+	    \vec v = x\vec b_1 + y\vec b_2
+	\]
+	and substitute for $\vec b_1$ and $\vec b_2$, we see this is equivalent to finding scalars $x,y$ so that
```
Comment (siefkenj) [inline problemsets-mat223/modules/module8.tex]: `, we see this` -> `. This`

Comment (siefkenj) [reply problemsets-mat223/modules/module8.tex]: `$x,y$` -> `$x$ and $y$`

File: problemsets-mat223/modules/module8.tex [hunk 5]
```diff
@@ -23,7 +23,30 @@
...
+	\[
+	    2\xhat-\yhat=x\xhat+(x+3y)\yhat.
+	\]
+	Solve the system
```
Comment (siefkenj) [inline problemsets-mat223/modules/module8.tex]: -> `Solving the system`

File: problemsets-mat223/modules/module8.tex [hunk 6]
```diff
@@ -23,7 +23,30 @@
...
+	\]
+	Solve the system 
+	\[
+	    \sysdelim\{.
+		\systeme{
+			x=2,
+			x+3y=-1
+		},
+	\]
+	We have $\vec v=2\vec b_1 - \vec b_2$. That is,
```
Comment (siefkenj) [inline problemsets-mat223/modules/module8.tex]: `We` -> `we`

File: problemsets-mat223/modules/module8.tex (hunk 3 above)
Comment (PINGXIANG) [reply problemsets-mat223/modules/module8.tex]: Ohhhh I see. I thought it is modest

File: problemsets-mat223/modules/module8.tex [hunk 7]
```diff
@@ -23,7 +23,30 @@
...
+	    [\vec v]_{\mathcal E}=\mat{2\\-1}.
+	\]
+	To find $[\vec v]_{\mathcal B}$, we can use a system of equations to find $\vec v$ as a linear combination of $\vec b_1$ and $\vec b_2$. That is, set 
+	\[
+	    \vec v = x\vec b_1 + y\vec b_2
+	\]
+	and substitute for $\vec v$, $\vec b_1$, and $\vec b_2$. This is equivalent to finding scalars $x$ and $y$ so that
+	\[
+	    \vec v = 2\xhat-\yhat=x(\xhat+\yhat)+3y\yhat=x\xhat+(x+3y)\yhat.
+	\]
```
Comment (siefkenj) [inline problemsets-mat223/modules/module8.tex]: This is getting very close to finished, but it doesn't flow right. Read the whole thing as it is and make the sentences fit together a bit better.

Comment (PINGXIANG) [reply problemsets-mat223/modules/module8.tex]: Ohhh I actually try to do that. But eventually I delete all of them and do as you said. Yeah I will do this

**Pull request merged**